src/askem_beaker/contexts/dataset/context.py

```diff
@@ -40,9 +40,12 @@ async def set_datasets(self, dataset_map, parent_header={}):
             elif isinstance(dataset_map_item, dict):
                 dataset_id = dataset_map_item["id"]
             else:
-                raise ValueError("Unable to parse dataset mapping")
-            meta_url = f"{os.environ['DATA_SERVICE_URL']}/datasets/{dataset_id}"
-            dataset_info_req = requests.get(meta_url)
+                raise ValueError("Unable to parse dataset mapping")      
+            meta_url = f"{os.environ['HMI_SERVER_URL']}/datasets/{dataset_id}"
```

Comment:
Do we not need to change the docker-compose.yaml environment.DATA_SERVICE_URL -> HMI_SERVER_URL for this?
I may be misunderstanding

Reply:
Ah so I updated the `env.example` and for development ive just used staging/beta to confirm that thing were working as expected but for true local end to end development we need to update the `docker-compose.yaml`.

When I'm running it I just use a `docker-compose-dev.yaml` like:

```
---
# Jupyter LLM docker compose by Matthew Printz

version: '3'
services:
  jupyter:
    build:
      dockerfile: Dockerfile
      context: .
    ports:
      - "8888:8888"
    environment:
      - DEBUG=1
      - JUPYTER_SERVER=http://jupyter:8888
      - JUPYTER_TOKEN=89f73481102c46c0bc13b2998f9a4fce
      - ENABLE_USER_PROMPT=true
      - OPENAI_API_KEY
    env_file:
      - .env      
    volumes:
      - ./src/askem_beaker:/home/jupyter/askem_beaker
      - .:/jupyter
    command: ["python", "-m", "beaker_kernel.server.dev", "watch"]
```

I could add this to the repo if helpful